package-lock.json

```diff
@@ -5029,7 +5029,8 @@
         "ansi-regex": {
```

Comment:
Please can the package-lock file changes be removed, this should not be in the PR.

It is possible you are using a slightly different NPM version, maybe try `nvm use` and then `npm install` (note nvm first). either way, just remove this file from the PR.